.github/issue_template.md

```diff
@@ -1,10 +1,10 @@
-# If your issue looks like hardware fault or a configuration problem this may be is the wrong place to post to.
-## Please consider using [RCGroups](https://rcgroups.com/forums/showthread.php?t=2464844), [Slack](https://slack.betaflight.tech) or any other forum before posting here or even better ask the manufacturer of the hardware you are using.
+# If your issue looks like a hardware fault or a configuration problem please don't raise an issue here.
+## Please consider using other user support options such as asking the manufacturer of the hardware you are using, [RCGroups](https://rcgroups.com/forums/showthread.php?t=2464844), [Slack](https://slack.betaflight.tech) or other user support forums & groups (e.g. facebook).
 
-## If you post here please follow these steps
-1.) Describe your problem
-2.) Include ways to reproduce the problem
-2.) Provide information about your flightcontroller and other components
-3.) Add the used configuration and firmware version
-4.) Create a diff and post it here in a code block (Simply put \`\`\` at the start and end of the diff block)
+## If you believe there is an issue with the firmware itself please follow these steps:
+1.) Describe your problem,
```

Comment:
@mikeller depends on how you would construct the sentence if you removed the dot points. If the items are independent lists then you would use semi colons. 

As an example (taken from: https://english.stackexchange.com/questions/75591/semicolons-at-the-end-of-list-items)

```The things that we use most commonly are:
- bats,
- balls, and
- wickets.```
Here, the list can be written as a single sentence.

```The things that we use most commonly are: bats, balls, and wickets.```

versus, that which include independent clauses, as in:

```I have visited many and many places throughout the world:
    - Sweden, Norway and Iceland in the Europe;
    - every single country of the Americas; 
    - also, China, Vietnam, Thailand and some more neighbouring countries in the far east. 
```

which as a single sentence would be written as:

```I have visited many and many places throughout the world: Sweden, Norway and Iceland in the Europe; every single country of the Americas; also, China, Vietnam, Thailand and some more neighbouring countries in the far east.```

See: https://owl.english.purdue.edu/owl/resource/607/04/

But in all honesty... its no one will notice :) and so long as it gets the message across! :)